foreman.fc

```diff
@@ -69,3 +69,6 @@
 
 /usr/share/gems/gems/foreman-tasks-.*/bin/foreman-tasks -- gen_context(system_u:object_r:foreman_tasks_exec_t,s0)
 /opt/theforeman/tfm/root/usr/share/gems/gems/foreman-tasks-.*/bin/foreman-tasks -- gen_context(system_u:object_r:foreman_tasks_exec_t,s0)
+
+# Red Hat Cloud plugin
+/var/lib/foreman/red_hat_inventory/uploads/uploader.sh -- gen_context(system_u:object_r:foreman_uploader_bin_t,s0)
```

Comment:
Before we add this, I'd like to investigate alternatives. This morning with @ShimShtein and @adamruzicka we talked about this. Other complications that we may run into is that `/var/lib` might be mounted `noexec` in which case it'd also fail.

Reply:
What would be the best location for it? It needs to be a place where it makes sense (something like `/usr/libexec`) but at the same time is writable by the rails process.

Reply:
My first question is why it needs to be dynamically generated.

Reply:
@ekohl, it's for disconnected systems. The idea is to generate a script that can be downloaded alongside with the payload and executed with minimal user interaction. For this reason I "hard code" some variables into the script, so the user will not need to input them.

Reply:
I had the very same question, you can still parametrize a short shell script as needed.

I agree that proper solution is to make this a generic script in `/usr/lib/libexec`.